ci/scripts/execute_redis_tests.sh

```diff
@@ -19,12 +19,14 @@
 
 cd ..
 
-# We are currently using a personal fork for this repo because our code does not implement all
+# We are currently using a patched version of this repo because our code does not implement all
 # Redis commands.  Once all commands needed to run relevant test files are implemented, we hope to
-# use Redis's repo instead.
-git clone --config transfer.fsckObjects=false https://github.com/prettyClouds/redis.git
+# use Redis's repo without a patch.
+git clone --config transfer.fsckObjects=false https://github.com/redis/redis.git
+cp geode-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch redis
 cd redis
 git checkout tests-geode-redis
```

Comment:
Correct me if I'm missing something, but I don't think this branch is in the `redis` repo.  I guess we could check out the 5.0 branch.  Have you tried this sequence locally?